R/ggExtra.R

```diff
@@ -9,4 +9,4 @@
 #'
 #' @docType package
 #' @name ggExtra
-NULL
+NULL
```

Comment:
can you set your editor to not remove newlines at end of files?

Reply:
I removed these lines manually. Some of the files had blank lines at the EOF while others didn't, so I was trying to make it consistent. Do you prefer to have blank lines at EOF?

Reply:
blank line at EOF is preferred. I don't actively go around different files adding it, but I prefer not to make any files go the opposite direction